perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -496,6 +496,35 @@ public UserExtSource call(ApiCaller ac, Deserializer parms) throws PerunExceptio
 		}
 	},
 
+	/*#
+	 * Gets list of all user's external sources with attributes. If attrsNames is empty
+	 * return all attributes. If attrsNames is null, this methods returns all attributes.
+	 *
+	 * @param user int User <code>id</code>
+	 * @return List<UserExtSource> list of user's external sources with attributes
+	 */
+	/*#
+	 * Gets list of all user's external sources with specified attributes.
```

Comment:
You should mention the possibility about null value for ``attrsNames`` here. It will return all attributes with non-null value. Empty list will return no attributes.